### PR TITLE
add case activity to cases page

### PR DESCRIPTION
**Pull request by MarkThomasLite**
add recent activity column to cases page
-enhanced cases templates to include extra data from the api

compared getting data via api using several queries and this appeared to be the quickest when run on local environment.
We needed last 2 audit activities per case but the case_id could be an action object or target object. Alternatives tried:

using union to get correct results from the fb
using filter to get last 2 activities per action and target (total 4) and filtering down to 2
a few different methods of massaging the data after retrieving results

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 exclude: (exporter/assets/built/|caseworker/assets/built/)
 default_language_version:
-  python: python3.7
+  python: python3.9
```
Comment (depsiatwal) [inline .pre-commit-config.yaml:3]: was this python version intended could cause issues for some dev may be worth having a separate PR for this

Comment (MarkThomasLite) [reply .pre-commit-config.yaml]: restored, I use this config locally as the 3.7 interpreter run during pre-commit has stopped working the past couple weeks

File: caseworker/templates/includes/case-row.html [hunk 2]
```diff
@@ -57,6 +57,49 @@
...
+							{{ update.user.first_name }} {{ update.user.last_name }} {{ update.text|linebreaksbr }}
+						</div>
+						{% if activity.additional_text %}
+							<div class="app-updates__activity">
+								{{ update.additional_text }}
+							</div>
+						{% endif %}
+					</li>
+					<li class="app-updates__item">
+						<div class="govuk-hint govuk-!-margin-0 govuk-!-margin-bottom-2">
```
Comment (kevincarrogan) [inline caseworker/templates/includes/case-row.html:94]: Is `govuk-hint` really the class we want to be using here? It's not really a hint.

**Review by kevincarrogan — APPROVED** [review]

**Review by kevincarrogan — APPROVED** [review]

**Review by saruniitr — APPROVED** [review]
